src/Simplex/Chat.hs

```diff
@@ -172,6 +173,11 @@ processChatCommand = \case
     asks agentAsync >>= readTVarIO >>= \case
       Just _ -> pure CRChatRunning
       _ -> startChatController user $> CRChatStarted
+  SetFilesFolder filesFolder' -> withUser' $ \_ -> do
```

Comment:
```suggestion
  SetFilesFolder filesFolder' -> withUser' $ \_ -> do
```

shouldn't it be `withUser`? Folder should be set once it's started, right?

Reply:
correct